feature-libs/asm/styles/components/_asm-main-ui.component.scss

```diff
@@ -196,3 +196,10 @@
     }
   }
 }
+
+.cx-asm-help-portal {
+  padding-inline-end: 10px;
+  .fa-circle-question {
```

Comment:
the name of 'fa-circle-question' is strange, what's the fa?

Reply:
define the icon named help, which is sourced from fontawesome's far fa-circle-question